pkg/namesgenerator/names-generator.go

```diff
@@ -431,6 +434,9 @@ var (
 		// Ian Murdock - founder of the Debian project - https://en.wikipedia.org/wiki/Ian_Murdock
 		"murdock",
 
+		//John Nash - father of Game Theory
```

Comment:
Can you;

- add a link to his wikipedia page; https://en.wikipedia.org/wiki/John_Forbes_Nash_Jr.
- nit: add a space before `John`
- perhaps use his full name, and mention that he was a  mathematician